posthog/migrations/0159_property_defs_search.py

```diff
@@ -0,0 +1,65 @@
+# Generated by Django 3.1.12 on 2021-07-08 12:40
+
+import warnings
+
+from django.contrib.postgres.indexes import GinIndex
+from django.contrib.postgres.operations import TrigramExtension
+from django.db import migrations
+from django.db.migrations.operations.models import IndexOperation
+
+
+class TrigramExtensionSuppressExceptions(TrigramExtension):
+    def database_forwards(self, app_label, schema_editor, from_state, to_state) -> None:
+        try:
+            return super().database_forwards(app_label, schema_editor, from_state, to_state)
+        except BaseException as e:
```

Comment:
Do we know what the error class is going to be when we want to ignore the failure? If so, let's use that instead of capturing e.g. db connection being down or something else!